packages/wonder-blocks-popover/components/focus-manager.js

```diff
@@ -197,15 +205,29 @@ export default class FocusManager extends React.Component<Props> {
     };
 
     render() {
+        const {children} = this.props;
+
         return (
             <React.Fragment>
                 {/* first sentinel */}
                 <div
                     tabIndex="0"
                     onFocus={this.handleFocusPreviousFocusableElement}
                 />
-                >
-                <div ref={this.getComponentRootNode}>{this.props.children}</div>
+                <div ref={this.getComponentRootNode}>
+                    {this.props.initialFocusEnabled ? (
+                        // If enabled, the focus will be passed to a given
+                        // element inside the children (popover dialog)
+                        <InitialFocus
+                            initialFocusId={this.props.initialFocusId}
```

Comment:
Yes, the scenario happens when we create an uncontrolled Popover. This means, that if the Popover is controlled using the `opened` prop, we might not want to  steal focus from another element in the DOM tree. Do you think it would be better to use a different approach?

Reply:
In the controlled state couldn't we just not set the `initialFocusId` if we don't want to steal focus?